deployment/live/gcp/example-gcp/terragrunt.hcl

```diff
@@ -1,25 +1,22 @@
 terraform {
-  source = "${get_repo_root()}/deployment/modules/gcp/gcs"
+  source = "${get_repo_root()}/deployment/modules/gcp//example-gcp"
 }
 
 locals {
+  env         = path_relative_to_include()
   project_id = get_env("GOOGLE_PROJECT", "trillian-tessera")
   location   = get_env("GOOGLE_REGION", "us-central1")
-  base_name   = get_env("TESSERA_BASE_NAME", "example-gcp")
+  base_name   = get_env("TESSERA_BASE_NAME", "${path_relative_to_include()}-example-gcp")
```

Comment:
Can you use $env here instead of the path calculation? It'd be clearer for a human reading it, even if the result is the same.

Reply:
I assumed I couldn't, but rather unexpectedly it seems like you can (or at least `${local.env}` you can)...